pkg/reconciler/pipelinerun/resources/input_output_steps.go

```diff
@@ -17,45 +17,13 @@ limitations under the License.
 package resources
 
 import (
-	"path/filepath"
-
 	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
 	resourcev1alpha1 "github.com/tektoncd/pipeline/pkg/apis/resource/v1alpha1"
 )
 
-// GetOutputSteps will add the correct `path` to the output resources for pt
-func GetOutputSteps(outputs map[string]*resourcev1alpha1.PipelineResource, taskName, storageBasePath string) []v1beta1.TaskResourceBinding {
-	var taskOutputResources []v1beta1.TaskResourceBinding
-
-	for name, outputResource := range outputs {
-		taskOutputResource := v1beta1.TaskResourceBinding{
-			PipelineResourceBinding: v1beta1.PipelineResourceBinding{
-				Name: name,
-			},
-			Paths: []string{filepath.Join(storageBasePath, taskName, name)},
-		}
-		// SelfLink is being checked there to determine if this PipelineResource is an instance that
-		// exists in the cluster (in which case Kubernetes will populate this field) or is specified by Spec
-		if outputResource.SelfLink != "" {
-			taskOutputResource.ResourceRef = &v1beta1.PipelineResourceRef{
-				Name:       outputResource.Name,
-				APIVersion: outputResource.APIVersion,
-			}
-		} else if outputResource.Spec.Type != "" {
-			taskOutputResource.ResourceSpec = &resourcev1alpha1.PipelineResourceSpec{
-				Type:         outputResource.Spec.Type,
-				Params:       outputResource.Spec.Params,
-				SecretParams: outputResource.Spec.SecretParams,
-			}
-		}
-		taskOutputResources = append(taskOutputResources, taskOutputResource)
-	}
-	return taskOutputResources
-}
-
 // GetInputSteps will add the correct `path` to the input resources for pt. If the resources are provided by
```

Comment:
Suggest removing the line about resources being provided by previous tasks, since that functionality is being removed.